Example/Tests/Core/UserDefaultsTweakProviderTests.swift

```diff
@@ -31,45 +31,45 @@ class UserDefaultsTweakProviderTests: XCTestCase {
         anotherConfiguration.set(12.34, feature: "feature_1", variable: "variable_2")
         anotherConfiguration.set(42, feature: "feature_1", variable: "variable_1")
         
-        XCTAssertTrue(anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_1")!.value == 42)
-        XCTAssertTrue(anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_2")!.value == 12.34)
-        XCTAssertTrue(anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_3")!.value == true)
-        XCTAssertTrue(anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_4")!.value == "hello")
+        XCTAssertTrue(try! anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_1").value == 42)
+        XCTAssertTrue(try! anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_2").value == 12.34)
+        XCTAssertTrue(try! anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_3").value == true)
+        XCTAssertTrue(try! anotherConfiguration.tweakWith(feature: "feature_1", variable: "variable_4").value == "hello")
     }
     
     func testReturnsNilForTweaksThatHaveNoUserDefaultValue() {
-        let tweak = userDefaultsTweakProvider.tweakWith(feature: Features.uiCustomization, variable: Variables.displayRedView)
+        let tweak = try? userDefaultsTweakProvider.tweakWith(feature: Features.uiCustomization, variable: Variables.displayRedView)
         XCTAssertNil(tweak)
     }
     
     func testUpdatesValueForTweak_withBool() {
         userDefaultsTweakProvider.set(true, feature: "feature_1", variable: "variable_1")
-        let tweak = userDefaultsTweakProvider.tweakWith(feature: "feature_1", variable: "variable_1")
-        XCTAssertTrue(tweak!.value == true)
+        let tweak = try! userDefaultsTweakProvider.tweakWith(feature: "feature_1", variable: "variable_1")
+        XCTAssertTrue(tweak.value == true)
```

Comment:
You can use `XCTUnwrap` instead
```suggestion
        let tweak = try XCTUnwrap(userDefaultsTweakProvider.tweakWith(feature: "feature_1", variable: "variable_1"))
        XCTAssertTrue(tweak.value)
```